src/index.ts

```diff
@@ -694,10 +695,29 @@ class Datastore extends DatastoreRequest {
     );
   }
 
+  /**
+   * Gets the database id that all requests will be run against.
+   *
+   * @returns {string} The database id that the current client is set to that
+   *    requests will run against.
+   */
+  getDatabaseId(): string | undefined {
+    return this.options.databaseId;
+  }
+
   getProjectId(): Promise<string> {
     return this.auth.getProjectId();
   }
 
+  /**
+   * Gets the database id that all requests will be run against.
+   *
+   * @param {any} [reqOpts] The request operations to add the databaseId to
+   */
+  addDatabaseIdToRequest(reqOpts: any): void {
```

Comment:
this looks more like an internal implementation detail. Do we need this on the surface layer?

Reply:
I guess if we follow the plan on the second comment then this will not be needed on the surface layer anymore.